deploying.md

```diff
@@ -0,0 +1,37 @@
+# Deploying
+There are multiple ways of deploying the defrag API.
+## Using Podman
+Using Podman is probably the easiest method.
+
+### Using prebuilt containers
+1. Create a .env file as described in [configuration.md](configuration.md).
+2. Make sure you have Podman installed: `sudo zypper install podman`
+3. Run the container: `podman run --env-file /path/to/env/file -p 80:8000 registry.opensuse.org/home/karatekhd/defrag/containers/defrag:latest`. Replace the `/path/to/env/file` with the actual path tp your env file.
```

Comment:
Forwarding 80 to 8000 won't work on a rootless podman, so we might want to use another port instead of 80.

Reply:
That's true, I'll change it.

Reply:
Fixed in 2807307